win/CS/HandBrakeWPF/Services/Encode/Factories/EncodeTaskFactory.cs

```diff
@@ -251,10 +251,10 @@ private Video CreateVideo(EncodeTask job)
             if (job.VideoEncodeRateType == VideoEncodeRateType.AverageBitrate)
             {
                 video.Bitrate = job.VideoBitrate;
-                video.MultiPass = job.MultiPass;
-                video.Turbo = job.TurboAnalysisPass;
             }
 
+            video.MultiPass = job.MultiPass;
```

Comment:
I might be inclined to special case guard this as ABR or VP9.

This likely protects against the UI state being ticked while disabled.  I'm not 100% sure it will actually be unticked when the checkbox is hidden.

Reply:
Yes, libhb will sanitize the values. But if you want to checkbox to be unticked too when switching to constant quality if the encoder doesn't support 2-pass we can make it so.

Reply:
Ok, cool. That's fine then.